### PR TITLE
Add changelog entry for version 2.0.2.

### PR DESCRIPTION
**Pull request by mbarbella-chromium**
PTAL. List of commits: https://github.com/google/clusterfuzz/compare/v2.0.1...master

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,14 @@
 Change Log
 ==========
 
+## Version 2.0.2
+- Improved Syzkaller support.
+- Support narrower bisection for regression/fix ranges.
+- Improve Rust crash detection signatures.
+- Improved Android KASan support.
+- Batch datastore operations more aggressively.
+- Various bug fixes.
```
Comment (oliverchang) [inline CHANGELOG.md:10]: Can add a point on grouping algorithm improvement to catch more variants of crashes involving inline frames.

Comment (inferno-chromium) [reply CHANGELOG.md:10]: also add
"Enable entropic fuzzing strategy in libFuzzer"
"Test past crash regressions in corpus pruning task"

Comment (mbarbella-chromium) [reply CHANGELOG.md:13]: Done!

**Pull request merged**